playbooks/openshift-node/private/contrail_sanitize.yml

```
@@ -0,0 +1,64 @@
+---
+- name: Openning port 53
+  hosts: "{{ openshift_node_scale_up_group | default('oo_nodes_to_config') }}"
+  tasks:
+    - name: open udp port 53
+      command: iptables -I INPUT 4 -j ACCEPT -p udp --dport 53
+      when: openshift_use_contrail | default(false) | bool
+
+- name: Contrail wait for come up
+  hosts: oo_first_master
+  tasks:
+    - name: Waiting for pods to come up
+      wait_for: timeout=360
+      delegate_to: localhost
+      when: openshift_use_contrail | default(false) | bool
+
+- name: Contrail wait for pod check script
+  hosts: oo_first_master
+  tasks:
+    - name: Check Pods script to Master Node
+      command: /tmp/wait_for_pod.sh
+      when: openshift_use_contrail | default(false) | bool
+      ignore_errors: True
+
+- name: Contrail Restarting dnsmasq service
+  hosts: "{{ openshift_node_scale_up_group | default('oo_nodes_to_config') }}"
+  tasks:
+    - name: Restart dnsmasq
+      service: name="dnsmasq" state=restarted enabled=yes
```

```suggestion
      service:
        name: "dnsmasq"
        state: restarted
        enabled=yes
```

Please avoid using INI-style syntax - it makes diffs look more complicated